src/client/components/Profile/Messages/MessagesContainer.component.js

```diff
@@ -27,24 +27,26 @@ export default class MessagesContainer extends React.Component {
 
   getMessages() {
     const messages = this.sortMessages();
-
+    let renderedIds = [];
     const renderedMessages = messages.slice(0, this.state.limit).map(msg => {
-      return (
-        <MessageRow
-          agencies={this.props.agencies}
-          agencyActions={this.props.agencyActions}
-          groupActions={this.props.groupActions}
-          groupState={this.props.groupState}
-          key={msg.createdEpoch}
-          message={msg}
-          readAction={this.props.readAction}
-          deleteAction={this.props.deleteAction}
-          renewLoanAction={this.props.renewLoanAction}
-          userstatusState={this.props.userstatusState}
-        />
-      );
+      if (!renderedIds.includes(msg.commentId)) {// only render message, if not already rendered
```

Comment:
Jeg mener ikke man kan være sikker på at en besked har et commentId. Der findes forskellige typer beskeder og det er kun type-commentWasAdded, som indeholder commentId. Så der bør laves et tjek om commentId findes.

Reply:
Selvfølgelig. Det er nu tilføjet i seneste commit.

Reply:
Ved vi, at der kun kommer dobbel kommentarer, når kommentaren er af typen-commentWasAdded? For i den fix jeg har lavet, tjekkes der kun på beskeder der har en commentId. Besekder uden commentId bliver renderet uden tjek om de allerede er renderet.